modules/installation-aws-iam-user.adoc

```diff
@@ -32,6 +32,17 @@ AWS permissions and attach it to the user, this is not the preferred option.
 The cluster will not have the ability to grant additional credentials to
 individual components, so the same credentials are used by all components.
 ====
++
+[IMPORTANT]
```

Comment:
Reading this section I don't see where a token would be used here, although it's quite possibly my own ignorance in this regard. :) Checking the link on creating an IAM user in AWS I don't see reference to tokens, either.

Are we certain this section is relevant for creating an IAM user?

Reply:
I think it needs to be mentioned in this module, but it's more relevant in a later step.